framework/src/executioners/EigenExecutionerBase.C

```diff
@@ -120,6 +119,14 @@ EigenExecutionerBase::init()
   // normalize solution to make |Bx|=_eigenvalue, _eigenvalue at this point has the initialized value
   makeBXConsistent(_eigenvalue);
 
+  if ( _problem.getDisplacedProblem() != NULL )
+  {
+    _problem.getDisplacedProblem()->syncSolutions(*_problem.getNonlinearSystem().currentSolution(),
```

Comment:
so my understanding is that this function only copy the solutions of the undisplaced systems to the local solution (non-ghosted) of the displaced system, which explains why we need the following two lines.